Jenkinsfile

```diff
@@ -163,9 +163,9 @@ pipeline {
                             script {
                                 // If this build is not a pull request, run owasp scan on the distribution. Otherwise run incremental scan
                                 if (env.CHANGE_ID == null) {
-                                    sh 'mvn verify -q -B -Powasp-dist -DskipTests=true -DskipStatic=true -pl !$DOCS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                                    sh 'mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@directories dependency-check:check dependency-check:aggregate -q -B -Powasp-dist -DskipTests=true -DskipStatic=true -pl !$DOCS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```

Comment:
❓ Were there issues without the addition of `org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@directories`?

Reply:
yes, so basically I was trying to get a little more efficient here. running the Maven`verify` phase runs multiple plugins (potentially ones that were ran in previous phases). So here, I just wanted to run `dependency-check` but found that it depends on the directories maven plugin as well. So that one has to be executed first for `dependency-check` to get the highest base directory to use the root suppressions files throughout all the modules.